lib/ui/widgets/delivery_info_banner.dart

```diff
@@ -0,0 +1,93 @@
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'package:zachranobed/extensions/build_context_extensions.dart';
+
+import '../../common/constants.dart';
+import '../../common/helper_service.dart';
+import '../../models/canteen.dart';
+import '../../models/delivery.dart';
+import '../../models/user_data.dart';
+import '../../notifiers/delivery_notifier.dart';
+import 'button.dart';
+import 'donation_countdown_timer.dart';
+import 'info_banner.dart';
```

Comment:
Nit: I do not remember if we agreed on "package" imports, or keep it as optional? 🤔

Reply:
I don't think we agreed, but we all prefer absolute 😄